Modules/Utils/CLI/UpdateConfig.js

```diff
@@ -0,0 +1,26 @@
+const { writeJSONAtomic: writeFile } = require("fs-nextra");
+
+/**
+ * Updates the config.json
+ * @argument {CLI} cli CLI class instance. Can be polyfilled with { sharder }
+ * @argument {Object} configJSON config.json object
+ * @returns {Promise<Boolean|Error>}
+ */
+module.exports = async function UpdateConfig (cli, configJSON) {
```

Comment:
When not using `this`, please try to use fat arrow functions at all times `() => {}`